napari/conftest.py

```diff
@@ -744,15 +745,97 @@ def my_start(self):
     )
 
 
+with contextlib.suppress(ImportError):
+    from pytestqt.qtbot import QtBot
+
+    class OwnQtBot(QtBot):
```

Comment:
Can you add a docstring for this? I don't understand the need for this class — or why it's under a supress ImportError guard.

Reply:
Will add docstring 

> I don't understand the need for this class — or why it's under a supress ImportError guard.

Because of `headless` test. Same reason why all fixtures have Qt imports inside, not top-level one.